test-clients/src/main/java/com/hedera/services/bdd/spec/queries/meta/HapiGetTxnRecord.java

```diff
@@ -115,8 +115,11 @@
 	private Optional<Consumer<Map<AccountID, Long>>> debitsConsumer = Optional.empty();
 	private Optional<ErroringAssertsProvider<List<TransactionRecord>>> duplicateExpectations = Optional.empty();
 	private Optional<Integer> childRecordsCount = Optional.empty();
-	private Optional<Integer> childRecordNumber = Optional.empty();
-	private Optional<String> aliasKey = Optional.empty();
+
+	private static record ExpectedChildInfo(String aliasingKey) {
+	}
```

Comment:
Empty method can be removed as well

Reply:
This isn't a method, it's a `record` definition ;)

Reply:
oops, read it as TransactionRecord for some reason.